### ISSUE TITLE
Chapter won't print completely

### ISSUE THREAD

**Issue by jangossner** [sillsdev/ptx2pdf#1000]
When I print Heb 6 by itself, I get only the last three lines of the second of two footnotes that are in the chapter. When I print it with the chapter before or after, it seems to print correctly. I can print a single chapter before or after it, too.

I don't really speak Github so I don't know how you are supposed to be able to look at what I'm doing to try to find a solution. I see that I can add files so I've added the PTXPrint output, I think, and I'll paste the sfm text of Heb 6 after that.

Thanks for your help,
Jan Gossner
jan_gossner@sil.org

[ETRBible_TeamReadthrough-sglspaced2col-heavierfont_HEB_ptxp.pdf](https://github.com/user-attachments/files/17729890/ETRBible_TeamReadthrough-sglspaced2col-heavierfont_HEB_ptxp.pdf)

<\c 6
\s Godea nilï hedabomäia, nilïa gelola nea, beaha namolö dialea
\p
\v 1 Amai gabugida, nilïa Gelesoïe ado mala mi gäwila nabi amo amilahilä, nilïle asigi falagasela mala geloi säla ïase amo nabala moholöi afiadi. Ado mala mi gäwila olelese nabi amoda, mosöïe igau gadoimelë. Igau gadoi bu elö made sagamolöda, mala gowe ili gadoia. Nilïa mala galone made difolalo amoïe hamoi hamosa nefolalo amo amilahilä, asigi besula baagi ïmolö amoïege, Godema asigi afädë negela namolö amoïege,
\v 2 ödägi elö elöla gelo mala amoïege, odoama busagia naabo negelo mala amoïege, amase, baagoi nowemolö mala amoïege, namolöi namolöïe mogemolö mala amoïege, bu säla olelemolö mademelë.\f * \fr 6.2 \ft Damo gogida, ödägi elö elöla gelo mala amoïe säi dialea gasoda, Giligi adoea fädäla made säi. Dolö odoaea helësësaleada, udia dolö asigi afädë negei ödägi gelo amoïemu made säi gabula helësëla, Godeïe siigi nigime baai made namolöïe, Yuda ilia bëgëla hamolo mala dilo amo gadola ödäea nigime hëlo amoïege, säi dialeala helësëi.\f*
\v 3 Nilïa ulufadigila moholöi afiadi. Godea nilïma fu ïaseda, amala hamodelä.
\p
\v 4-6 Amaimo, udia dolö dafidi Godeïe yo digalomadi nelo amo fisila dïgïla obëgëi amoli bu asigi besumäia, abeahäla sigi obëgëmolödu. Amo ilia selesaia mosöala galu ï hëdä melï amoge, Eleloi Gesamige ilibalä nelo. Amase, ilia Godeïe ado mala hedabi gabulage, hobea namolö amoïe geloige melï nelo. Amo hamosa maadelahilä, ilia fisila obëgëi amoïe, ili asigi besumäia bu sigi obëgëmolö amo ulïmelë. Mida, udia dolö amolia Godeïe malö amo hilile naaboea i fügäigi bu baala negelahilä, udia dolö yafie neebea ami, e neala hewalasise amea.
\p
\v 7 Sege digaha aligise amoda, nagagi digalo. Naga amogi sagai amo gägä negese amoda, udia dolö sagalo amo hili neala nülo nea. Godeada naga amo neala hedabolasilo nea
\v 8 gaso, naga nafadegida, äyo gägäige giisi nafadege bäulo nea. Naga amo Godea nafademolömelëla bea sämolöda, ga made dialeamelë. Hobeafalïda, nulogi gobesa maademolömelë.
\p
\v 9 Ne damelë obeägïlola dili-e, walio nilïa, dilima ado gowe gelola säsisago. Gaso, nilïa dili helësësaleada, hädamade säi amo gadola made, dili hedabi nea, Godea dili damula awäla dofä sigi obëgëmolömelëla nilïa gelola helësësa neamelë.
\v 10 Godea mala mogoi madeda, made hamolo nea. Dilia e boba hamosa neebe amo ulufala olela, Geleso sosogo nefolalebe amoli ugila negesa nelo amola, walio ugila negesamu neebe amoge, dilia hamoi hamoi amoge, ea made elaboi nea.
\v 11 Nilïa edefademalä hamese amoda, dili sulubadelaea, Godea hobea afädë hamomolö gabula asigia sisiläfüsa neebe amo dofä hamola ulufamolö amoïe, dili hïdo hïdolaea Gode boba hamola, hamoi hamola wagamolöïe hamomäiala hamesamelë.
\v 12 Dilia hamoi made hamolo, habobame gadoi made namiäiala hamesamelë. Amala made neada, udia dolö odoaea asigi afädë negela, aso maadela neebe amo gadola aligise ameada, Godea ïmolömelëla säla negei dibe amo nelo amoli gadoi, dilige namiäiala hamesamelë.
\s Godea dofä hamomolömelëla säla negei dibe amo nilïa dofäla gelola helësëmolöi galea
\p
\v 13 Godea Abalahamama ïmolömelëla dafidi säla negemolö nea, dolö afädë Gode hï e ëi ilibia edefade neebe amo gadoi elö maade ameada, hïle ëigila migalïa diigila sälo amo gadola, gelola säla negesalea
\v 14 sähalea, Na di dofä obeägïlasila, dïe diaua mäu yafie dofä ïmolömelëla säla negei. \rq Mi Gäwi 22.16-17 \rq*
\v 15 Ameada, Abalahama ea fologola made, aso maadela nea, ïmäiala aumala beahamu nelo. Amala beaha nelo amea, Godea ïmolömelëla säla negei amo neala gaui.
\p
\v 16 Dolö afädëa nelebo afädë hamomolöïe ado geloi afädë säla negemolö neada, hï e gadoi made, dolö elö e baagila awäla, dafidilo ëi sälo neebe amoïe ëi säla, nelebo afädë hamomolö amo, migalïa diigila säla negelo. Amala säla negelo amoda, udia dolöea ado säla negei amo gadogo gidogola säla, made neadigimäiala helësëla, dolö amea amala sälo.
\v 17 Amalo amo gadola, Godea hamomolöla säla negei amo hobeala neala gaumolö amolima ea hamomolö helësëi amoda, bedefamolöi mademelëla fädäla olemolöïe, migalïa diigila sälo amo gadola wäla säla negei.
\v 18 Hä, Godea hamomolöla säla negei ameage, amase, ea migalïa diigila sälo amo gadola gelola säi ameage, nilïle iso asigi dofä gelolasimäia, äudä amole säla ï. Mida, bedefamolöi made äudä amolegida, Godea wäfalï sämolöi mademelë. Hä, emadi fisäla asi nilïa, asigia sisiläfüsa namolö amo fëdöla gaula nefolaleawemelë.
\v 19 Nilïa asigia sisiläfüsa nefolalebe weda, nelebo yöi igi aeanea hamoi amo dofä fologomolöi made amea, ödä duugi gide begelo amo gadola, nilïle iso asigigi begei diëgi wäla nefolalea. Asigia sisiläfüsa neebe amoda, gobela ïlo mosöa fagulo edefade gosai dilo amoïe baagidola, hogoli amoïe mumuso isogi nogila digaha nea.
\v 20 Hogoli amogida, hobeala nilï nogila digamäia, nilï ugila negemolöïe, Yesudale hogoli amogi nogila digai. Eda Melegesedege e nelo amo gadoi wäla nea, ea odala ï neala ïlo busa edefade namolöimu namolö amo nea.\f * \fr 6.20 \ft Gäheo i sulubadelaïe segeyo afädëgida, Isilale udia dolö nefolalo amolile odala ï neala ïlo busa edefade nelo ameada, hamoi edefade afädë hamomolö dilo. Eada, hä, nafade hëmäia odala ï neala ïmolöïe heale usi amo gaulahilä, Godeïe mosö Odala Ï Hogoli Aiai, Gode hï e namolömelëla hogoli säi amogi gaula nogila digaha aligilo. Gobela Ïlo 16ge, Hibulu 9.1-7ge bimalo. Amase walioda, Yesu e dofä odala ï neala ïlo busa edefade amo besela galai nea, Godeïe mosöa Hebeni neada, udia dolöea nafade hamose amoli hësa nea. Hibulu 2.17ge, 4.14ge, 8.2ge bimalo.\f*
>

**Comment by jangossner** [comment]
This chapter is printing properly now. IDK what made the difference.

**Comment by markpenny** [comment]
Thanks for letting us know that it is working again; but it would be good to figure out why it went wrong in the first place.

The next time you have such a problem, please click on the Help tab and use the "Create Archive..." button there to make a .zip file with all the needed components for us to replicate the issue. [I realize that this feature may also be broken in a recent version - so it is possible that you already tried this and it failed. We will make sure it is working properly in the upcoming release 2.6.11]

**Comment by jangossner** [comment]
Thanks for your reply, Mark, and for pointing me to the 'Create Archive...'
button. I will keep that in mind.

I was wondering what I should do with the archive it creates so I tried it
out to see if the program would tell me what to do. That is, do I create a
GitHub entry and attach it or what. Anyway, I got a notice that what I'm
pasting below had been copied to the clipboard. I suspect it is just
telling you what you already know, that it doesn't work. The zip file
created has no contents.

So, my question still remains, what does one do with the zip file?

I'm currently listening to your recent AI: Curse or Blessing presentation.
Very interesting. I've been using AI (David Baines first, then SF) to do
back translation for some of the remaining books in our NT project. It has
less than a full NT to learn on and so far the results are marginally time
saving, but probably more consistent and thorough. Sorry, rabbit trail.

God Bless,
Jan

<Mailto: ***@***.***>

Traceback (most recent call last):
  File "ptxprint\gtkview.py", line 3894, in onCreateZipArchiveClicked
  File "ptxprint\view.py", line 1740, in createArchive
  File "ptxprint\view.py", line 1770, in _archiveAdd
  File "ptxprint\view.py", line 1615, in _getArchiveFiles
TypeError: unsupported operand type(s) for +: 'Project' and 'str'

<class 'TypeError'>: unsupported operand type(s) for +: 'Project' and 'str'

PTXprint Version 2.6.10>

**Comment by markpenny** [comment]
Sorry that I wasn't more specific. You can either (a) send the resulting .zip archive file to ptxprint_support (at) sil.org or if it is too large to e-mail, (b) place the .zip file in [this Google Drive](https://drive.google.com/drive/folders/1S63WeFUg96ZdlA5IdS4E3AI7kC3HaxUo?usp=drive_link) folder, and I can pick it up from there.

BTW - [version 2.6.11](https://software.sil.org/downloads/r/ptxprint/SetupPTXprint(2.6.11).exe) has just been released, so you should be in good shape to use the "Create Archive..." button successfully from now on.

**Comment by jangossner** [comment]
Much appreciated, Mark.

I've updated the app and will send the archive to that eddress.

Jan

**Comment by markpenny** [comment]
Thanks you @jangossner for uncovering this bug. It has now been fixed and will be working as expected in 2.6.12 (which will hopefully be released in a few days).

By the way, when using Variable Line Spacing, I have found that the space between the text and the footnote area rule is rather minimal, giving it a very tight/busy look. It is easy to adjust that on the Notes+Refs tab using the values highlighted in yellow below:

![Image](https://github.com/user-attachments/assets/4ab77e9f-6b85-4bd0-b496-41663c052e53)

This will give you some more uniform space as seen here:
![Image](https://github.com/user-attachments/assets/a3b82c77-5f7a-4ac6-b700-31c783317904)

**Comment by jangossner** [comment]
You're welcome, Mark. I'm glad to give something back.

And thank you for the tip about footnote spacing.

God Bless,
Jan

**Comment by jangossner** [comment]
Evening all,

I decided to try to recreate this issue by going back and getting a copy of
Heb 6 from the time I had the problem of incomplete printing. I pasted the
text into my ETRve project since it didn't already have any text in it. I
successfully got an incomplete printing but when I went to create archive I
got a similar failure in creating the archive to what I got before that bug
was fixed in 2.6.11, it created an empty archive. The message it gave is
below. I'll also paste the text of that chapter in case that is at all
helpful to you.

I may try pasting that outdated text into my regular project, assuring I
can undo it, and see if that will re-create the partial printing issue.

JG

<Mailto: ***@***.***>

Traceback (most recent call last):
  File "ptxprint\gtkview.py", line 3894, in onCreateZipArchiveClicked
  File "ptxprint\view.py", line 1726, in createArchive
  File "ptxprint\runjob.py", line 303, in doit
  File "ptxprint\runjob.py", line 303, in <genexpr>
  File "ptxprint\runjob.py", line 540, in dojob
  File "ptxprint\runjob.py", line 687, in sharedjob
  File "ptxprint\texmodel.py", line 575, in asTex
  File "ptxprint\texmodel.py", line 1559, in generateImageCopyrightText
KeyError: 'rtl'

<class 'KeyError'>: 'rtl'

PTXprint Version 2.6.11>

Heb 6 from 14 Nov 24:
<HTML clipboard
\c 6
\s Godea nilï hedabomäia, nilïa gelola nea, beaha namolö dialea
\p \v 1 Amai gabugida, nilïa Gelesoïe ado mala mi gäwila nabi amo
amilahilä, nilïle asigi falagasela mala geloi säla ïase amo nabala moholöi
afiadi. Ado mala mi gäwila olelese nabi amoda, mosöïe igau gadoimelë. Igau
gadoi bu elö made sagamolöda, mala gowe ili gadoia. Nilïa mala galone made
difolalo amoïe hamoi hamosa nefolalo amo amilahilä, asigi besula baagi
ïmolö amoïege, Godema asigi afädë negela namolö amoïege, \v 2 ödägi elö
elöla gelo mala amoïege, odoama busagia naabo negelo mala amoïege, amase,
baagoi nowemolö mala amoïege, namolöi namolöïe mogemolö mala amoïege, bu
säla olelemolö mademelë.\f * \fr 6.2 \ft Damo gogida, ödägi elö elöla gelo
mala amoïe säi dialea gasoda, Giligi adoea fädäla made säi. Dolö odoaea
helësësaleada, udia dolö asigi afädë negei ödägi gelo amoïemu made säi
gabula helësëla, Godeïe siigi nigime baai made namolöïe, Yuda ilia bëgëla
hamolo mala dilo amo gadola ödäea nigime hëlo amoïege, säi dialeala helësëi.
\f* \v 3 Nilïa ulufadigila moholöi afiadi. Godea nilïma fu ïaseda, amala
hamodelä.
\p \v 4-6 Amaimo, udia dolö dafidi Godeïe yo digalomadi nelo amo fisila
dïgïla obëgëi amoli bu asigi besumäia, abeahäla sigi obëgëmolödu. Amo ilia
selesaia mosöala galu ï hëdä melï amoge, Eleloi Gesamige ilibalä nelo.
Amase, ilia Godeïe ado mala hedabi gabulage, hobea namolö amoïe geloige
melï nelo. Amo hamosa maadelahilä, ilia fisila obëgëi amoïe, ili asigi
besumäia bu sigi obëgëmolö amo ulïmelë. Mida, udia dolö amolia Godeïe malö
amo hilile naaboea i fügäigi bu baala negelahilä, udia dolö yafie neebea
ami, e neala hewalasise amea.
\p \v 7 Sege digaha aligise amoda, nagagi digalo. Naga amogi sagai amo gägä
negese amoda, udia dolö sagalo amo hili neala nülo nea. Godeada naga amo
neala hedabolasilo nea \v 8 gaso, naga nafadegida, äyo gägäige giisi
nafadege bäulo nea. Naga amo Godea nafademolömelëla bea sämolöda, ga made
dialeamelë. Hobeafalïda, nulogi gobesa maademolömelë.
\p \v 9 Ne damelë obeägïlola dili-e, walio nilïa, dilima ado gowe gelola
säsisago. Gaso, nilïa dili helësësaleada, hädamade säi amo gadola made,
dili hedabi nea, Godea dili damula awäla dofä sigi obëgëmolömelëla nilïa
gelola helësësa neamelë. \v 10 Godea mala mogoi madeda, made hamolo nea.
Dilia e boba hamosa neebe amo ulufala olela, Geleso sosogo nefolalebe amoli
ugila negesa nelo amola, walio ugila negesamu neebe amoge, dilia hamoi
hamoi amoge, ea made elaboi nea. \v 11 Nilïa edefademalä hamese amoda, dili
sulubadelaea, Godea hobea afädë hamomolö gabula asigia sisiläfüsa neebe amo
dofä hamola ulufamolö amoïe, dili hïdo hïdolaea Gode boba hamola, hamoi
hamola wagamolöïe hamomäiala hamesamelë. \v 12 Dilia hamoi made hamolo,
habobame gadoi made namiäiala hamesamelë. Amala made neada, udia dolö
odoaea asigi afädë negela, aso maadela neebe amo gadola aligise ameada,
Godea ïmolömelëla säla negei dibe amo nelo amoli gadoi, dilige namiäiala
hamesamelë.
\s Godea dofä hamomolömelëla säla negei dibe amo nilïa dofäla gelola
helësëmolöi galea
\p \v 13 Godea Abalahamama ïmolömelëla dafidi säla negemolö nea, dolö afädë
Gode hï e ëi ilibia edefade neebe amo gadoi elö maade ameada, hïle ëigila
migalïa diigila sälo amo gadola, gelola säla negesalea \v 14 sähalea, Na di
dofä obeägïlasila, dïe diaua mäu yafie dofä ïmolömelëla säla negei. \rq Mi
Gäwi 22.16-17 \rq* \v 15 Ameada, Abalahama ea fologola made, aso maadela
nea, ïmäiala aumala beahamu nelo. Amala beaha nelo amea, Godea ïmolömelëla
säla negei amo neala gaui.
\p \v 16 Dolö afädëa nelebo afädë hamomolöïe ado geloi afädë säla negemolö
neada, hï e gadoi made, dolö elö e baagila awäla, dafidilo ëi sälo neebe
amoïe ëi säla, nelebo afädë hamomolö amo, migalïa diigila ado gelola säla
negelo. Amala säla negelo amoda, udia dolöea ado säla negei amo gadogo
gidogola säla, made neadigimäiala helësëla, dolö amea amala sälo. \v 17
Amalo amo gadola, Godea hamomolöla säla negei amo hobeala neala gaumolö
amolima ea hamomolö helësëi amoda, bedefamolöi mademelëla fädäla olemolöïe,
migalïa diigila ado gelola sälo amo gadola wäla säla negei. \v 18 Hä, Godea
hamomolöla säla negei ameage, amase, ea migalïa diigila sälo amo gadola
gelola säi ameage, nilïle iso asigi dofä gelolasimäia, äudä amole säla ï.
Mida, bedefamolöi made äudä amolegida, Godea wäfalï sämolöi mademelë. Hä,
emadi fisäla asi nilïa, asigia sisiläfüsa namolö amo fëdöla gaula
nefolaleawemelë. \v 19 Nilïa asigia sisiläfüsa nefolalebe weda, nelebo yöi
igi aeanea hamoi amo dofä fologomolöi made amea, ödä duugi gide begelo amo
gadola, nilïle iso asigigi begei diëgi wäla nefolalea. Asigia sisiläfüsa
neebe amoda, gobela ïlo mosöa fagulo edefade gosai dilo amoïe baagidola,
hogoli amoïe mumuso isogi nogila digaha nea. \v 20 Hogoli amogida, hobeala
nilï nogila digamäia, nilï ugila negemolöïe, Yesudale hogoli amogi nogila
digai. Eda Melegesedege e nelo amo gadoi wäla nea, ea odala ï neala ïlo
busa edefade namolöimu namolö amo nea.\f * \fr 6.20 \ft Gäheo i
sulubadelaïe segeyo afädëgida, Isilale udia dolö nefolalo amolile odala ï
neala ïlo busa edefade nelo ameada, hamoi edefade afädë hamomolö dilo.
Eada, hä, nafade hëmäia odala ï neala ïmolöïe heale usi amo gaulahilä,
Godeïe mosö Odala Ï Hogoli Aiai, Gode hï e namolömelëla hogoli säi amogi
gaula nogila digaha aligilo. Gobela Ïlo 16ge, Hibulu 9.1-7ge bimalo. Amase
walioda, Yesu e dofä odala ï neala ïlo busa edefade amo besela galai nea,
Godeïe mosöa Hebeni neada, udia dolöea nafade hamose amoli hësa nea. Hibulu
2.17ge, 4.14ge, 8.2ge bimalo.\f*>

**Comment by jangossner** [comment]
I did <try pasting that outdated text into my regular project, assuring I
can undo it, and see if that will re-create the partial printing issue.> I
did, it didn't.

I did see that an earlier version of the chapter had an extraneous \p
marker after the \v 11 marker and before the text of the verse, and another
one at the end of the chapter, following a long footnote. Putting those in
again didn't recreate the issue and I'm going to leave it at that with
trying to recreate the partial chapter printing.

IDK if the partial printing and the failure to create an archive in my
ETRve project is anything for you to follow up on, or not.

Jan
